scripts/deploy-suite.js

```diff
@@ -158,7 +158,7 @@ async function main(env, facetConfig) {
   // LENS
   transactionResponse = await bosonConfigHandler.setAuthTokenContract(
     AuthTokenType.Lens,
-    authTokenContracts.lensAddress,
+    process.env.LENS_ADDRESS || authTokenContracts.lensAddress,
```

Comment:
Can you please make this change in function `getAuthTokenContracts()` (at the top of the file) instead of here?
And the same goes for `process.env.ENS_ADDRESS`.

Reply:
changed, thanks!